### PR TITLE
Clarify GPU support

### PR DESCRIPTION
**Pull request by znation**

### CODE REVIEW

**Comment by avlaskin** [comment]
That was instant, thanks!

File: README.md [hunk 1]
```diff
@@ -132,13 +132,13 @@ GPU Support
 
 Turi Create **does not require a GPU**, but certain models can be accelerated 9-13x when utilizing a GPU. 
 
-Turi Create automatically utilizes Mac GPUs for the following tasks (requires macOS 10.13+):
+Turi Create automatically utilizes Mac GPUs for the following tasks (requires macOS 10.14+):
 * Image Classification
```
Comment (srikris) [inline README.md]: GPU acceleration on IC works on 10.13 too.

File: README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Version 5.0 (Beta)
 -----------
 
 Turi Create 5.0 -- now in beta -- includes:
-* GPU Acceleration on Macs (10.14+)
+* GPU Acceleration on Macs for Image Similarity (10.13+), Image Classification (10.13+), Object Detection (10.14+), and Activity Classification (10.14+).
```
Comment (gustavla) [inline README.md]: Is it clear this is a macOS version?

Comment (znation) [reply README.md]: Maybe not, but it's starting to get really verbose. Can you think of a better way to express this nuance?

File: README.md [hunk 3]
```diff
@@ -132,13 +132,13 @@ GPU Support
 
 Turi Create **does not require a GPU**, but certain models can be accelerated 9-13x when utilizing a GPU. 
 
-Turi Create automatically utilizes Mac GPUs for the following tasks (requires macOS 10.13+):
-* Image Classification
-* Object Detection
-* Activity Classification
-* Image Similarity
+Turi Create automatically utilizes Mac GPUs for the following tasks:
+* Image Classification (requires macOS 10.13+)
```
Comment (srikris) [inline README.md]: You can remove requires

**Pull request merged**